plugin.video.srgssr_ch_replay/addon.py

```diff
@@ -282,7 +302,7 @@ def _add_show(name, url, urn, mode, desc, iconimage, channel, numberOfEpisodes):
     liz = xbmcgui.ListItem(name)
     liz.setLabel2(desc)
     liz.setArt({'poster': iconimage, 'banner': iconimage, 'fanart': iconimage, 'thumb': iconimage})
-    #TODO setInfo is deprecated (see comments https://github.com/xbmc/repo-plugins/pull/3722)
+    #TODO setInfo might become deprecated (see comments https://github.com/xbmc/repo-plugins/pull/3722)
```

Comment:
@enen92 I checked the add-on with the new Nexus build and so far I do not see any issues. I will re-test it again when it's closer to the release.

Reply:
Yeah it still works, it's just deprecated. Probably removed in v21